<?xml version="1.0"?>

<launch>
    
    <arg name="robot_name" default="sergio"/>
    
    <!-- Loads robot specific parameters for the robot_skills to the parameter server -->
    <!--<param command="load" file="$(find robot_skills)/parameters/$(arg robot_name).yaml" />-->
    <rosparam command="load" file="$(find robot_skills)/parameters/sergio.yaml" ns="/sergio/skills"/>

</launch>
